conf/app.conf

```diff
@@ -0,0 +1,7 @@
+appname = goMarketer
+httpport = 8080
+runmode = dev
```

Comment:
Not sure if more config is needed but the address is blank when running the app with `go run main.go`

![image](https://user-images.githubusercontent.com/696529/101602899-652a1f80-3a31-11eb-8041-36ab8e26de31.png)

Reply:
I fixed it in https://github.com/andyduong1920/go_marketer/pull/1/commits/7008584bc0439448cc02593b7c209c98bfb32617

![image](https://user-images.githubusercontent.com/11751745/101617084-c4dcf680-3a42-11eb-92ca-62e4778bac7e.png)